modyn/models/articlenet/articlenet.py

```diff
@@ -43,6 +43,9 @@ def __call__(self, data: torch.Tensor) -> torch.Tensor:
         # of the entire input sequence
         return pooled_output
 
+    def _reorder_cache(self, past: Any, beam_idx: Any) -> None:
+        pass
```

Comment:
ah yeah, when you merge main, you can remove this again. we just disabled the pylint error here, not sure why it is complaining here

Reply:
Done